crates/rolldown/tests/esbuild/dce/package_json_side_effects_false_keep_bare_import_and_require_es6/bypass.md

```diff
@@ -36,7 +36,6 @@ var init_demo_pkg = __esm({ "node_modules/demo-pkg/index.js"() {
 //#endregion
 //#region src/entry.js
 init_demo_pkg();
-init_demo_pkg();
```

Comment:
Removing is expected. `Users/user/project/node_modules/demo-pkg/index.js` is marked with `sideEffect: fasle`. We have the same result as esbuild.